tests/utils/verification/volumeproperties.go

```diff
@@ -150,3 +150,13 @@ func VerifyDetachedStatus(name, hostName, esxName string) bool {
 	log.Printf("Timed out to poll status\n")
 	return false
 }
+
+// GetPolicyNameForVol returns the vsan policy name used by the volume using docker cli
+func GetPolicyNameForVol(hostname string, volName string) string {
```

Comment:
I don't think we should move this function to test/utils/dockercli.  This function is used to get some volume attribute, which is the same as existing function in this file.